pyigm/utils.py

```diff
@@ -411,3 +411,60 @@ def get_components(obj,ion,zrange=None):
         complist = [comp for comp in obj._components if
                  ((comp.Zion==Zion)&(comp.zcomp>zrange[0])&(comp.zcomp<zrange))]
     return complist
+
+def confintervals(hits,totals,sig=1):
+    """Wrapper for confinterval() function to accept arrays of hits and totals
+    
+    Parameters
+    ----------
+    hits : list or array
+        Numbers of instances
+    totals : list or array
+        Total numbers of instances possible
+    sig : int
+        Confidence level in multiples of sigma
+
+    Returns
+    -------
+    fracs : list of floats
+        Fractions of hits/totals
+    lolims : list of floats 
+        Lower bounds of confidence interval
+    uplims : list of floats
+        Upper bounds of confidence interval
+    """
+    fracs=np.zeros(len(hits)) ; uplims=np.zeros(len(hits)) ; lolims=np.zeros(len(hits))
+    for i,tt in enumerate(totals):
+        fracs[i],lolims[i],uplims[i]=confinterval(hits[i],totals[i],sig)
+    return fracs,lolims,uplims
+
+def confinterval(hits,total,sig=1):
+    """ Calculate Wilson confidence intervals
+    Parameters
+    ----------
+    hits : int
+        Number of instances
+    total : int
+        Total number of instances possible
+    sig : int
```

Comment:
same query as above.  It is possible someone
will want a 95% c.l. or a 99% c.l. and therefore
would insert a float.

Reply:
Ah, I see below that you are translating to conf interval anyhow.
I might then suggest that one can provide either (and none defaults
to 1 sigma equivalent).   We'd then need to use a method to translate
sigma to conf which I am sure I have somewhere..  Either in pyigm or
linetools..

Reply:
Done!  I couldn't find a preexisting method, so I added a simple one.